app/controllers/countries_controller.rb

```diff
@@ -7,13 +7,18 @@ def index
     @countries
   end
 
+  def show
```

Comment:
we would probably need to start adding ~controller~ requests specs

Reply:
https://rspec.info/features/6-0/rspec-rails/request-specs/request-spec/#:~:text=Request%20specs%20provide%20a%20thin,if%20you%20have%20set%20config.